src/sentry/constants.py

```diff
@@ -232,6 +234,41 @@ def get_all_languages():
 
 NATIVE_UNKNOWN_STRING = '<unknown>'
 
+# to go from a marketing page slug like /for/android/ to the integration id
+# (in _platforms.json), for looking up documentation urls, etc.
+MARKETING_SLUG_TO_INTEGRATION_ID = {
+    # java
+    "java": "java",
+    "kotlin": "java",
+    "scala": "java",
+    "android": "java-android",
+
+    # TODO: add more...
+}
+
+# ???
+# via Eric: sdk name -> "server/product/event pipeline" slug
+# via Eric: python.flask -> whatever you'd send in the `integration` field in the events
```

Comment:
Yeah it's just manually coded in each SDK, so they'd have to sync up with the names they use in `sentry-doc-config.json`.

But we can make a lookup function on the Sentry side regardless. I understand the need/purpose better now.